Automattic/wp-calypso
# A4A > Migrations: Add steps for WordPress.com & Pressable self migration tool.

Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/1488

## Proposed Changes

This PR:

- Adds the self migration steps for WordPress.com.
- Adds the self migration steps for Pressable.
- Adds contact form support for newly added pages
- Adds event tracking for the buttons
- Fixes a typo
- Makes an update to the tool to fold the card when a step is marked as done or already completed. 

## Why are these changes being made?

* To add steps for WordPress.com & Pressable self migration tool.

## Testing Instructions

1) Open the A4A live link.
2) Go to the /migrations/overview page > Click the `Migrate your site` dropdown button
3) Select the `Self migrate to WordPress.com` option > Verify that the steps are matching as per the design(link in the issue). Also, verify that the cards fold when they are marked as done and opens when reset. Clicking on the contact us button on the page heading should open the contact us modal with the appropriate product selected. 
4) Repeat step 2 > Verify the same for `Self migrate to Pressabel` option.

![screencapture-agencies-localhost-3000-migrations-overview-migrate-to-wpcom-2024-11-19-11_52_32](https://github.com/user-attachments/assets/04e63f98-61b7-4e37-80e0-a6ec69c3ae2c)

![screencapture-agencies-localhost-3000-migrations-overview-migrate-to-pressable-2024-11-19-11_54_16](https://github.com/user-attachments/assets/a262512a-5253-41a8-9838-e5bcccb31373)

@madebynoam: Please check if the mobile view ok. 

<img width="364" alt="Screenshot 2024-11-19 at 12 01 12 PM" src="https://github.com/user-attachments/assets/14185b67-23e6-468f-891d-88e3d59a2cb4">

Mixed state:

<img width="1728" alt="Screenshot 2024-11-19 at 11 51 54 AM" src="https://github.com/user-attachments/assets/859a42d5-dd03-4c4e-83b9-213be0b3ef2d">

## Pre-merge Checklist
